djoser/utils.py

```diff
@@ -21,13 +20,15 @@ def decode_uid(pk):
 
 
 def login_user(request, user):
-    token, _ = authtoken.models.Token.objects.get_or_create(user=user)
+    Token = import_string(settings.get('TOKEN_MODEL'))
```

Comment:
It would be awesome if you could remove duplicate definition of token (exactly same thing is happening at `logout_user`). My proposition for solution for this problem is to move it to the top of the file like:

```
from . import settings

Token = import_string(settings.get('TOKEN_MODEL'))
```
That way we will also avoid including another PEP-8 violation into djoser (local variable names should be all lowercase).